packages/model/addon/-private/model.js

```diff
@@ -821,6 +717,19 @@ class Model extends EmberObject {
     return this._internalModel.transitionTo(name);
   }
 
+  notifyPropertyChange(key) {
```

Comment:
sure. We're doing this because we manage tags for many properties, this is how we ensure those tags are notified when appropriate, not only by our own code but by external code as well.